tests/test-update-direct.c

```diff
@@ -246,6 +246,110 @@ test_cancel_update (EosUpdaterFixture *fixture,
   g_assert_cmpuint (helper.cancelled_error_count, ==, helper.cancel_calls_count);
 }
 
+static void
+update_version_state_changed_cb (EosUpdater *updater,
+                                 GParamSpec *pspec,
+                                 gpointer data)
+{
+  EosUpdaterState state = eos_updater_get_state (updater);
+  GMainLoop *loop = (GMainLoop *) data;
+
+  if (state == EOS_UPDATER_STATE_POLLING)
+    return;
+
+  g_main_loop_quit (loop);
+}
+
+static void
+test_update_version (EosUpdaterFixture *fixture,
```

Comment:
Could you add a comment above the test function explaining what the test does and what it’s trying to test, please?

Reply:
Right! Sorry I forgot this. I had a bunch of code already from last week, so I forgot to update that part.